boltdesignsystem/bolt
# Hotfix: correct spacing for non-link navbar title

## Jira

http://vjira2:8080/browse/WWWD-4030

## Summary

Fixes a spacing regression for non-link navbar title.

## Details

1. Minimal code change to CSS to fix the spacing
2. Created consistent markup between linked and non-link titles

## How to test

Run the branch locally and check the demo page, make sure the spacing between the icon and the title is the same whether the title is linked or not.

* Linked title: http://localhost:3000/pattern-lab/?p=components-navbar-linked-title
* Non-link title: http://localhost:3000/pattern-lab/?p=components-navbar
